frontend/src/components/Agreements/AgreementDetails/AgreementHistoryList.js

```diff
@@ -11,17 +11,30 @@ const findObjectTitle = (historyItem) => {
     }
 };
 
+const logItemTitle = (historyItem) => {
+    const className = convertCodeForDisplay("className", historyItem.class_name);
+    if (historyItem.event_type === "NEW") {
+        return `New ${className} Created`;
+    } else if (historyItem.event_type === "UPDATED") {
+        return `${className} Updated`;
+    } else if (historyItem.event_type === "DELETED") {
+        return `${className} Deleted`;
+    }
+    return `${className} ${historyItem.event_type}`;
+};
+
 const summaryMessage = (historyItem) => {
     const className = convertCodeForDisplay("className", historyItem.class_name);
+    const classNameLower = className.toLowerCase();
+    const classNameSentence = classNameLower.charAt(0).toUpperCase() + classNameLower.slice(1);
     const userFullName = historyItem.created_by_user_full_name;
```

Comment:
also we have a custom hook `useGetUserFullNameFromId.js`  but may not be applicable here